src/azure-cli/azure/cli/command_modules/reservations/_help.py

```diff
@@ -162,3 +162,84 @@
     type: string
     short-summary: Id of the subscription to look up applied reservations
 """
+
+helps['reservations reservation-order calculate'] = """
+type: command
+short-summary: calculate price
+long-summary: calculate price for reservation
+parameters:
+  - name: --applied-scope-type
+    type: string
+    short-summary: choose Single or Shared as appliedscope
```

Comment:
For this parameter, allowed values are Single and Shared, right?
If yes, please use [get_enum_types()](https://github.com/Azure/azure-cli/blob/8adf97355a605d147afc26f6c1b6f6afb6651c2f/doc/authoring_command_modules/authoring_commands.md#registering-enums) and refine your help message.

Reply:
Have made it enum in params.py

Reply:
It looks your short-summary doesn't work. May I know you why add additional description here?
![image](https://user-images.githubusercontent.com/49508232/69246580-ec68e300-0be3-11ea-8e0f-113a3edb2d26.png)